In house patch to modify dependencies openstackclient's requirements files.
The specific reasons are as follows:

    cliff-tablib:  Not applicable
    python-cinderclient:  push to version 1.3.1

--- python-openstackclient-1.0.4/requirements.txt.orig	2015-11-19 12:56:09.037137656 -0700
+++ python-openstackclient-1.0.4/requirements.txt	2015-11-19 12:51:38.972729254 -0700
@@ -6,7 +6,6 @@ six>=1.9.0
 
 Babel>=1.3
 cliff>=1.10.0,<1.11.0  # Apache-2.0
-cliff-tablib>=1.0
 oslo.config>=1.9.3,<1.10.0  # Apache-2.0
 oslo.i18n>=1.5.0,<1.6.0  # Apache-2.0
 oslo.utils>=1.4.0,<1.5.0                       # Apache-2.0
@@ -14,7 +13,7 @@ oslo.serialization>=1.4.0,<1.5.0
 python-glanceclient>=0.15.0,<0.18.0
 python-keystoneclient>=1.1.0,<1.4.0
 python-novaclient>=2.22.0,<2.24.0
-python-cinderclient>=1.1.0,<1.2.0
+python-cinderclient>=1.1.0,<=1.3.1
 python-neutronclient>=2.3.11,<2.5.0
 requests>=2.2.0,!=2.4.0
 stevedore>=1.3.0,<1.4.0  # Apache-2.0

--- python-openstackclient-1.0.4/python_openstackclient.egg-info/requires.txt.orig	2015-11-19 12:56:59.337766773 -0700
+++ python-openstackclient-1.0.4/python_openstackclient.egg-info/requires.txt	2015-11-19 12:57:13.594181649 -0700
@@ -2,7 +2,6 @@ pbr>=0.6,!=0.7,<1.0
 six>=1.9.0
 Babel>=1.3
 cliff>=1.10.0,<1.11.0  # Apache-2.0
-cliff-tablib>=1.0
 oslo.config>=1.9.3,<1.10.0  # Apache-2.0
 oslo.i18n>=1.5.0,<1.6.0  # Apache-2.0
 oslo.utils>=1.4.0,<1.5.0                       # Apache-2.0
@@ -10,7 +9,7 @@ oslo.serialization>=1.4.0,<1.5.0
 python-glanceclient>=0.15.0,<0.18.0
 python-keystoneclient>=1.1.0,<1.4.0
 python-novaclient>=2.22.0,<2.24.0
-python-cinderclient>=1.1.0,<1.2.0
+python-cinderclient>=1.1.0,<=1.3.1
 python-neutronclient>=2.3.11,<2.5.0
 requests>=2.2.0,!=2.4.0
 stevedore>=1.3.0,<1.4.0  # Apache-2.0

